skstore/ts/src/prv/skstore_skjson.ts

```diff
@@ -267,17 +277,30 @@ function clone<T>(value: T): T {
 }
 
 interface FromWasm extends WasmAccess {
-  SKIP_SKJSON_startCJObject: () => ptr;
-  SKIP_SKJSON_addToCJObject: (obj: ptr, name: ptr, value: ptr) => void;
-  SKIP_SKJSON_endCJObject: (obj: ptr) => ptr;
-  SKIP_SKJSON_startCJArray: () => ptr;
-  SKIP_SKJSON_addToCJArray: (arr: ptr, value: ptr) => void;
-  SKIP_SKJSON_endCJArray: (arr: ptr) => ptr;
-  SKIP_SKJSON_createCJNull: () => ptr;
-  SKIP_SKJSON_createCJInt: (v: int) => ptr;
-  SKIP_SKJSON_createCJFloat: (v: float) => ptr;
-  SKIP_SKJSON_createCJString: (str: ptr) => ptr;
-  SKIP_SKJSON_createCJBool: (v: boolean) => ptr;
+  SKIP_SKJSON_startCJObject: () => ptr<Internal.VectorPairStringCJSON>;
```

Comment:
Given that this (and similarly for VectorCJSON a few lines down) are only used here, could we drop those specialized definitions from skstore_internal_types and just define aliases here? opened a PR against your branch showing what I mean

Reply:
You're right, I'll do that

Reply:
done in https://github.com/mbouaziz/skdb/pull/1 if that looks alright to you :)